src/compileApplicationToWasm.js

```diff
@@ -4,11 +4,12 @@ import { mkdir, readFile } from "node:fs/promises";
 import { isFile } from "./isFile.js";
 import { isFileOrDoesNotExist } from "./isFileOrDoesNotExist.js";
 import wizer from "@bytecodealliance/wizer";
+import weval from "@cfallin/weval";
```

Comment:
A standard technique for lazy initialization in JavaScript would be to turn this into a dynamic `import()` nested into the async code path where Weval is used only.